Singularity Build gramtools issue

Hello:
The singularity image build command  fails at the point of installing gramtools below is a snapshot of the command and error messages:

singularity build clockwork_container.img clockwork_container.def

________________________ gramtools _________________________#
pip3 install --process-dependency-links wheel git+https://github.com/iqbal-lab-org/gramtools@9313eceb606a6fc159e4a14c168b7a6f888c5ed2
+ pip3 install --process-dependency-links wheel git+https://github.com/iqbal-lab-org/gramtools@9313eceb606a6fc159e4a14c168b7a6f888c5ed2
Collecting git+https://github.com/iqbal-lab-org/gramtools@9313eceb606a6fc159e4a14c168b7a6f888c5ed2
  Cloning https://github.com/iqbal-lab-org/gramtools (to 9313eceb606a6fc159e4a14c168b7a6f888c5ed2) to /tmp/pip-041uygn9-build
  Could not find a tag or branch '9313eceb606a6fc159e4a14c168b7a6f888c5ed2', assuming commit.
DEPRECATION: Dependency Links processing has been deprecated and will be removed in a future release.
Requirement already satisfied: wheel in /usr/lib/python3/dist-packages
Collecting cluster_vcf_records>=0.9.2 (from gramtools==1.6.0)
  Downloading https://files.pythonhosted.org/packages/74/36/4285d072b4ac0d16ec278b1fe386ced0313cd8888fa90d26d2562b58f049/cluster_vcf_records-0.13.2.tar.gz
Collecting py-cortex-api>=1.0 (from gramtools==1.6.0)
  Downloading https://files.pythonhosted.org/packages/8a/0b/c81cf0f0feeda39cc42e7551864635ac911a3312406d3d68e836b1058a5e/py-cortex-api-2.2.0.tar.gz
Requirement already satisfied: pyvcf>=0.6.8 in /usr/local/lib/python3.6/dist-packages (from gramtools==1.6.0)
Requirement already satisfied: scipy>=1.0.1 in /usr/local/lib/python3.6/dist-packages (from gramtools==1.6.0)
Collecting bitarray (from cluster_vcf_records>=0.9.2->gramtools==1.6.0)
  Downloading https://files.pythonhosted.org/packages/57/3d/db2c679f5343f7bb79ccd7366441d9e872f60c8e499e15b95551dce9a5ea/bitarray-2.0.0.tar.gz (70kB)
    100% |################################| 71kB 6.5MB/s 

.........

.........

SUCCESS: sdsl was installed successfully!
    The sdsl include files are located in '/tmp/pip-041uygn9-build/cmake-build-release/libgramtools/include'.
    The library files are located in '/tmp/pip-041uygn9-build/cmake-build-release/libgramtools/lib'.
    
    Sample programs can be found in the examples-directory.
    A program 'example.cpp' can be compiled with the command:
    g++ -std=c++11 -DNDEBUG -O3 [-msse4.2] \
       -I/tmp/pip-041uygn9-build/cmake-build-release/libgramtools/include -L/tmp/pip-041uygn9-build/cmake-build-release/libgramtools/lib \
       example.cpp -lsdsl -ldivsufsort -ldivsufsort64
    
    Tests in the test-directory
    A cheat sheet in the extras/cheatsheet-directory.
    Have fun!
    [ 28%] No install step for 'sdsl'
    [ 29%] No test step for 'sdsl'
    [ 30%] Completed 'sdsl'
    [ 30%] Built target sdsl
    Makefile:83: recipe for target 'all' failed
    make: *** [all] Error 2
    ERROR: gramtools backend compilation returned:  2
    
    ----------------------------------------
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-041uygn9-build/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-o6jmmrip-record/install-record.txt --single-version-externally-managed --compile" failed with error code 255 in /tmp/pip-041uygn9-build/
FATAL:   While performing build: while running engine: exit status 1

--- a/python/clockwork/tests/ena_downloader_test.py
+++ b/python/clockwork/tests/ena_downloader_test.py
@@ -86,7 +86,7 @@ class TestEnaDownloader(unittest.TestCase):
         )
         self.assertEqual("SAMEA2533482", got_sample)
         self.assertEqual("Illumina HiSeq 2500", got_instrument)
-        self.assertEqual("FZB", got_center_name)
+        self.assertEqual("FORSCHUNGSZENTRUM BORSTEL", got_center_name)
 
     def test_write_import_tsv(self):
         """Test _write_import_tsv"""

--- a/scripts/install_dependencies.sh
+++ b/scripts/install_dependencies.sh
@@ -188,7 +188,7 @@ pip3 install python-dateutil requests pysam pyfastaq pymysql numpy openpyxl pyfl
 
 
 #________________________ gramtools _________________________#
-pip3 install --process-dependency-links wheel git+https://github.com/iqbal-lab-org/gramtools@9313eceb606a6fc159e4a14c168b7a6f888c5ed2
+pip3 install --process-dependency-links wheel git+https://github.com/iqbal-lab-org/gramtools@8af53f6c8c0d72ef95223e89ab82119b717044f2
 
 #________________________ mummer ____________________________#
 cd $install_root

